go-controller/pkg/ovn/egressip.go

```diff
@@ -1007,6 +1007,12 @@ func (oc *Controller) addPodEgressIPAssignments(name string, statusAssignments [
 		if err != nil {
 			return nil
 		}
+		// Retry if LSP is scheduled for deletion
```

Comment:
This comment is a little confusing(We are not doing any explicit retry here, just returning).
Could you also add a longer comment explaining why we are checking if expires is not zero(just like in the commit message)?  It might became be useful once we want to reuse the cache in the same way for something else in the future.

Reply:
I mean the handler update triggers the retry :D 
sure